src/CloudModelGenerator/CodeGenerator.cs

```diff
@@ -57,7 +59,8 @@ public void GenerateTypeProvider()
 
         private void SaveToFile(string content, string fileName)
         {
-            string outputPath = _outputDir + $"{fileName}.cs";
+            string suffix = string.IsNullOrWhiteSpace(_transformFilenamesText) ? ".cs" : $".{_transformFilenamesText}.cs";
```

Comment:
ok. since named formatting parameters (like the mentioned "{fileName}.mytext" or "{FileName}.mytext") are not supported in .net without using an external library and string interpolation is no good in this case, let's stick with suffixes..